index.html

```diff
@@ -12,7 +12,16 @@
 <main>
   <header>
     <nav>
-      <div class="logo"><p>John Doe</p></div>
+      <div class="logo"><p>Tessy</p></div>
+      <ul class="menu-bar">
+        <li><a href="#frame" class="head-link">Portfolio</a></li>
+        <div class="indicator"></div>
+        <li><a href="#about" class="head-link">About</a></li>
+        <div class="indicator"></div>
+        <li><a href="#form-cont" class="head-link">Contact</a></li>
+        <div class="indicator"></div>
+      <li><a href="#" class="head-link icon-mess"><img src="./images/icon-mess.png" alt="main-image"></a></li>
+      </ul>
       <div class="hamburger">
         <span class="bar"></span>
         <span class="bar"></span>
```

Comment:
- [ ] As one of the goal of this project is to follow figma design as close as possible, Please make sure our header section is correctly aligned and follows the figma template, currently your font-size is not correct and is too big for your header links, also it should be aligned to the right see your design :point_down: 

![image](https://user-images.githubusercontent.com/97521557/206560285-ad6e3b4d-c4fc-488d-a447-ca44055f6ca6.png)

see figma design :point_down: 

![image](https://user-images.githubusercontent.com/97521557/206560376-8ebd2cdb-6fe1-4bdb-903b-bfa32db1902c.png)